assets/js/index.js

```diff
@@ -186,36 +166,80 @@ $(document).ready(function () {
     );
   });
 
+  //  Text Copy-Select functionality
+  function copy(textToCopy, $element) {
+    //temp input
+    var input = $('<input type="text" value=\'' + textToCopy + "' />");
+    input.appendTo("body");
+    input.get(0).select();
+    input.get(0).setSelectionRange(0, 99999); /*For mobile devices*/
+    document.execCommand("copy");
+    //remove temp input
+    input.remove();
+    //hide tooltip
+    setTimeout(() => $element.tooltip("hide"), 1500);
+  }
+
+  /**
+   * Copy tooltip functionality
+   * @param {object} copiedTooltipOptions
+   */
+  // Copied tooltip functions
+  var copiedTooltipOptions = {
+    title: "Copied",
+    trigger: "click",
+    placement: "bottom",
+  };
+
   $(".button-caption-sub").tooltip(copiedTooltipOptions);
 
   $(".button-caption-sub").click(function () {
     var classes = $(this).text().trim();
     classes = classes.replace(/\./g, "");
-
     copy(classes, $(this));
   });
 
   $(".script-copy").tooltip(copiedTooltipOptions);
 
   $(".script-copy").click(function () {
     var script = $(this).text().trim();
-
     copy(script, $(this));
   });
 
-  $(".sidebar-toggler").on("click", function (e) {
-    e.stopPropagation();
-    toggleSidebar();
+  // Theme changer button
+  $(".toggle-theme").on("click", function () {
+    toggleTheme();
   });
 
-  //Closes sidebar if screen size is less than 768 pixels
-  $(".sidebar a").on("click", function () {
-    var w = document.body.clientWidth;
-    if (w < 768) {
-      toggleSidebar();
+  // Download Button button
+  $(
+    "#downloadGithubRawHeader, #downloadGithubRawButton, #downloadGithubRawHowTo"
+  ).on("click", downloadGithubCssRaw);
+
+  function downloadGithubCssRaw() {
+    var link =
+      "https://cdn.statically.io/gh/shahednasser/sbuttons/c135f5f7/dist/sbuttons.min.css";
```

Comment:
change link from "shahednasser/sbuttons" to "sButtons/sbuttons" as we changed it to organization

Reply:
Done in recent [PR](https://github.com/sButtons/sbuttons/pull/583/commits/2b0f376df56af6a20f2cc3207aa8de4e84a8d511).

Reply:
Is this issue still unresolved? I still got red icon notification.